tests/unit/amo/sagas/test_home.js

```diff
@@ -171,7 +172,43 @@ describe(__filename, () => {
       expect(expectedAction).toEqual(loadAction);
     });
 
-    it('does not fetch featured themes if includeFeaturedThemes is false', async () => {
+    it('does not fetch featured themes if includeTrendingExtensions is false', async () => {
+      const collections = [];
+
+      const featuredExtensions = createAddonsApiResult([fakeAddon]);
+      mockSearchApi
+        .expects('search')
+        .returns(Promise.resolve(featuredExtensions));
+
+      const featuredThemes = createAddonsApiResult([fakeTheme]);
+      mockSearchApi.expects('search').returns(Promise.resolve(featuredThemes));
+
+      const popularExtensions = createAddonsApiResult([fakeAddon]);
+      mockSearchApi
+        .expects('search')
+        .returns(Promise.resolve(popularExtensions));
+
+      _fetchHomeAddons({
+        collectionsToFetch: [],
+        includeTrendingExtensions: false,
+      });
+
+      const loadAction = loadHomeAddons({
+        collections,
+        shelves: {
+          featuredExtensions,
+          featuredThemes,
+          popularExtensions,
+          trendingExtensions: null,
+        },
+      });
+
+      const expectedAction = await sagaTester.waitFor(loadAction.type);
+      mockSearchApi.verify();
+      expect(expectedAction).toEqual(loadAction);
+    });
+
+    it('does not fetch trending extensions if includeFeaturedThemes is false', async () => {
```

Comment:
I believe this should be "does not fetch *featured themes* if includeFeaturedThemes is false".